### DISCUSSION TITLE
npm run lint:css NOT WORKING!

### DISCUSSION THREAD

**Discussion by edualfaro** [lab-antwerp-1/home#69] [Q&A]
Dear all,
I have tried to pass "**npm run lint:css**" but there is no way. I have even modified the "**package.json**" and write there the **direct path to the file "styles.css" inside the styles folder**, but not working. Maybe it is because I am doing under windows, but I cannot explain why it is not working.

Do you experience something similar with this "npm run lint:css"? Any idea why this is happening?

I was thinking that maybe npm is taking its own folder as starting point, so it could not find any *.css of course, but looking at the log file, seems to be doing properly:

---
![image](https://user-images.githubusercontent.com/87333985/137098280-e808e629-d860-46f5-8503-91139cda9350.png)
---

Find attached an "snapshoot" of my screen for your reference.
As you can see, I have al the files, the path seems to be ok, but it is not working. Thank you so much!

![image](https://user-images.githubusercontent.com/87333985/137096918-e5944ce0-a65e-4cb3-a605-a0b2276509e0.png)

**Comment by colevandersWands** [comment]
@edualfaro, this might be a Windows compatibility issue.  it looks like you're on windows?  I haven't been able to reproduce your error on my computer

Can you share your question on Slack to make sure others see it? I'm curious who else has had this problem

**Comment by tahminarasoli** [comment]
Hi, @edualfaro I have the same problem. how did you solve this one can you please help me with this?

![error](https://user-images.githubusercontent.com/12840564/138130629-ec67dab9-2dd3-4ccf-b6b3-9d86452cfa32.PNG)

**Reply by edualfaro** [reply]
Yes, **I finally solved by installing Linux Ubuntu** in my computer. As you can see, with Linux there is not a problem at all, all lint checks will pass properly. If you want to install Linux OS in order to try before creating partitions, dual boot... you can use a Virtual Machine like [https://www.virtualbox.org](https://www.virtualbox.org) from Oracle. It is free to install and you can install Linux Ubuntu ISO there [https://ubuntu.com/download/desktop](https://ubuntu.com/download/desktop)
![image](https://user-images.githubusercontent.com/87333985/138144643-a0690042-2735-4939-9280-5f89fc57f9aa.png)
